### DISCUSSION TITLE
Setting Chrome (and other apps) Notifications to be always Float

### DISCUSSION THREAD

**Discussion by ghost** [hyprwm/Hyprland#7054] [Q&A]
Hi,

Is there a way to make hyprland set notification pop-ups to always be float (or ignore the current tiling layout)?

**Comment by prateekamana** [comment]
bump

**Comment by koknese** [comment]
bump

**Comment by vaxerski** [comment]
windowrules?

**Comment by n-connect** [comment]
Here's a window rule I use to make sure BT mgmt GUI always float:
`windowrule = float,class:^(blueman-manager)$`
Same exist for many others like that, needed some more for e-mail client's new mail etc.. Checking with `hyprctl clients` Chrome has this class: "google-chrome"

Although I'm using a notification app separately (dunst, mako, lately swaync -> seems the mosr feature rich one), so notifications from any app: browser, mail, terminal always the same -> the notification solution pops-up.
Test link: https://www.bennish.net/web-notifications.html

**Comment by voidzero** [comment]
the notifications i'm seeing have no identifiers, they're essentially empty class, empty title. So I just tried matching those, but i haven't tried it for long enough to see if this will affect other eventual windows with empty properties if there are any.

```
hl.window_rule({
	name = "float-empty-wayland-popups",
	match = {
		class = "^$",
		title = "^$",
		xwayland = false,
		fullscreen = false,
		pin = false,
	},
	float = true,
	move = "monitor_w-380 34",
})
```
